homeassistant/components/plugwise/climate.py

```diff
@@ -130,7 +130,7 @@ def hvac_action(self) -> HVACAction | None:
             return HVACAction.IDLE
 
         hc_data = self.coordinator.data.devices[
-            self.coordinator.data.gateway["heater_id"]
+            self.coordinator.data.gateway["heater_id"]  # type: ignore [index]  # typing through #92253
```

Comment:
Again, you should be able to refactor this in a preliminary PR (as opposed to a follow-up PR)